include/mbedtls/mbedtls_config.h

```diff
@@ -1335,6 +1335,15 @@
  */
 //#define MBEDTLS_RSA_NO_CRT
 
+/**
+ * \def MBEDTLS_RSA_MIN_KEY_SIZE
+ *
+ * Minimum RSA key size allowed in bits.
+ *
+ * Minimum possible value is 128 bits.
+ */
+#define MBEDTLS_RSA_MIN_KEY_SIZE 1024
```

Comment:
IIRC numerical values need to go into the “General configuration options” section, due to how `config.py` works. I'm not sure if that's why `test_psa_crypto_config_accel_rsa_signature` is failing.

Reply:
Added it, Thank you this was the reason CI was failing.